Laboratory/Tests/L0-Parsing/Declarations/Identifiers/IdentifiersStartingWithKeywords-pass.ele

```diff
@@ -0,0 +1,2 @@
+constraintBlah = 5
+structFoo = 10
```

Comment:
intrinsic functionZoop?

Reply:
Added more tests for `intrinsic` and all the valid combinations